pkg/systemd/generate/containers.go

```diff
@@ -182,6 +184,7 @@ func generateContainerInfo(ctr *libpod.Container, options entities.GenerateSyste
 		CreateCommand:     createCommand,
 		RunRoot:           runRoot,
 		containerEnv:      envs,
+		IdentifySpecifier: false,
```

Comment:
There is no need to set this, the default is false.

Reply:
DONE :+1: